platform/vs/onie.mk

```diff
@@ -1,4 +1,7 @@
 ONIE_RECOVERY_IMAGE = onie-recovery-x86_64-kvm_x86_64-r0.iso
 $(ONIE_RECOVERY_IMAGE)_URL = "https://sonicstorage.blob.core.windows.net/packages/onie/onie-recovery-x86_64-kvm_x86_64-r0.iso?sv=2015-04-05&sr=b&sig=XMAk1cttBFM369CMbihe5oZgXwe4uaDVfwg4CTLT%2F5U%3D&se=2155-10-13T10%3A40%3A13Z&sp=r"
-
-SONIC_ONLINE_FILES += $(ONIE_RECOVERY_IMAGE)
+ONIE_RECOVERY_MKVM_4ASIC_IMAGE = onie-recovery-x86_64-mkvm_x86_64_4_asic-r0.iso
+$(ONIE_RECOVERY_MKVM_4ASIC_IMAGE)_URL=""
+ONIE_RECOVERY_MKVM_6ASIC_IMAGE = onie-recovery-x86_64-mkvm_x86_64_6_asic-r0.iso
+$(ONIE_RECOVERY_MKVM_6ASIC_IMAGE)_URL=""
+SONIC_ONLINE_FILES += $(ONIE_RECOVERY_IMAGE) $(ONIE_RECOVERY_MKVM_4ASIC_IMAGE) $(ONIE_RECOVERY_MKVM_6ASIC_IMAGE)
```

Comment:
I see the ONIE_RECOVERY_IMAGE used in the mk file --> platform/vs/kvm-image.mk. Is that used ?
platform/vs/kvm-image.mk:$(SONIC_KVM_IMAGE)_FILES = $(ONIE_RECOVERY_IMAGE)

Reply:
Updated platform/vs/kvm-image.mk, it is getting used, without this update the new recovery iso files will not get downloaded.